biggerquery_tutorial/part1.ipynb

```diff
@@ -198,6 +198,27 @@
     "all_transactions_from_single_day.peek('2019-01-01')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Estimating the cost of query\n",
+    "The estimate_costs method returns string with estimated number of bytes read by the query and estimated cost of query in dollars."
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "costs = dataset.estimate_costs('''\n",
```

Comment:
It won't work. You didn't add estimate_costs to interactive layer.

Reply:
Done